src/page/summary-step.page.ts

```diff
@@ -4,7 +4,7 @@ export class SummaryStepPage {
   private proceedToCheckoutButton: ElementFinder;
 
   constructor () {
-    this.proceedToCheckoutButton = $('.cart_navigation span');
+    this.proceedToCheckoutButton = $('.cart_navigation > .button > span');
```

Comment:
try to find a better locator, something that uses the name or title attribute for example, in that way, we can understand what element you are referring